public/javascripts/running.js

```diff
@@ -168,39 +148,34 @@ function enableInteractive() {
 }
 
 function disableInteractive() {
-    window.hide("stop_button");
+    $("#stop_button").hide();
     sendCommand("disable_interactive_mode");
 }
 
 function sendCommand(command) {
-    var wid = window.testStatus.workerid;
+    var wid = testStatus.workerid;
```

Comment:
http://stackoverflow.com/questions/1144043/benefit-of-using-window-prefix-in-javascript has more  leaves very little bonus points on using windows. 

I agree with your consistency argument, but I dislike unneeded scope calls - so my vote would be to remove windows. everywhere